core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -518,6 +518,8 @@ var ExplorationEditorMainTab = function() {
   this.expectContentToMatch = async function(richTextInstructions) {
     await waitFor.visibilityOf(
       stateContentDisplay, 'State content display not showing up');
+    // Wait for any text to be present to reduce flakes.
+    await action.waitForNonEmptyText('Elem', stateContentDisplay);
```

Comment:
What is the significance of the word 'Elem'? Can/should it be a named constant?

Reply:
It's the element name, I think "State content display" is more appropriate here.